src/ucs/memory/numa.c

```diff
@@ -147,7 +147,7 @@ ucs_numa_node_t ucs_numa_node_of_device(const char *dev_path)
     long parsed_node;
     ucs_status_t status;
 
-    status = ucs_read_file_number(&parsed_node, 0, "%s/numa_node", dev_path);
+    status = ucs_read_file_number(&parsed_node, 1, "%s/numa_node", dev_path);
```

Comment:
it's better to add some print on the line 154 then

Reply:
but it does not print anything when silent arg is provided

Reply:
Yes. I meant to update `ucs_read_file_vararg`, and add else case to print the message with other verbosity level.

Reply:
it's also an option, but i'd check where it is used to avoid trace flooding